inc/core/templates/theme-notification-banner.php

```diff
@@ -0,0 +1,258 @@
+<?php
+/**
+ * Theme admin functions.
+ *
+ * @package BCGov Block Theme
+ */
+
+namespace Bcgov\Theme\Block;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit( '<h1>Direct access denied.</h1>' );
+}
+
+if ( ! current_user_can( 'manage_options' ) ) {
+	exit( '<h1>You do not have sufficient permissions to edit this page.</h1>' );
+}
+
+$notice_enabled             = esc_attr( get_option( 'notice_enabled' ) );
+$notice_homepage_only       = esc_attr( get_option( 'notice_homepage_only' ) );
+$notice_label_bold          = esc_attr( get_option( 'notice_label_bold' ) );
+$notice_label_big           = esc_attr( get_option( 'notice_label_big' ) );
+$notice_status              = esc_attr( get_option( 'notice_status' ) );
+$notice_content             = esc_attr( get_option( 'notice_content' ) );
+$notice_button_enabled      = esc_attr( get_option( 'notice_button_enabled' ) );
+$notice_button_label        = esc_attr( get_option( 'notice_button_label' ) );
+$notice_button_aria_label   = esc_attr( get_option( 'notice_button_aria_label' ) );
+$notice_button_link         = untrailingslashit( esc_attr( get_option( 'notice_button_link' ) ) );
+$notice_button_utm_campaign = esc_attr( get_option( 'notice_button_utm_campaign' ) );
+
+?>
+
+
+<div class="bcgov-block-theme-page-header">
```

Comment:
i wonder if there is a way to optimize setting html markup, to use reusable code.